test/e2e/common/setup-kubestellar.sh

```diff
@@ -135,6 +133,61 @@ wait-for-cmd "(kubectl --context '$HOSTING_CONTEXT' -n wds1-system wait --for=co
 
 echo "transport controller is running."
 
+# --- Start of ArgoCD Smoke Test ---
```

Comment:
A test does not belong in the setup script. You can include it in a testing script/program.